apps/osim/tests/conftest.py

```diff
@@ -58,15 +57,6 @@ def test_api_uri_osidb(test_scheme_host_osidb, api_version):
     return f"{test_scheme_host_osidb}/api/{api_version}"
 
 
-@pytest.fixture
-def command_curl():
-    """define path to curl"""
-    test_curl_path = get_env("TEST_CURL_PATH")
-    if test_curl_path is not None:
-        return test_curl_path
-    return "/usr/bin/curl"
```

Comment:
why? the whole point of integration tests is to test the application from outside, which we don't do in the CI yet btw so might as well not touch this

Reply:
makes sense, I will leave it as-is